src/hooks/screen/useChatScreenHook.ts

```diff
@@ -3,6 +3,7 @@ import 'react-native-get-random-values';
 
 import * as React from 'react';
 import SimpleToast from 'react-native-simple-toast';
+import {FlatList} from 'react-native';
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The import of `FlatList` from 'react-native' is added but it's not clear why this was necessary as the old hunk did not include this. Please ensure that this import is actually used in the code.
```diff
+ import {FlatList} from 'react-native';
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->